tests/parser/one-line-enum-no-comma-ending.jakt

```diff
@@ -0,0 +1 @@
+enum Fruit { Apple }
```

Comment:
this is valid Jakt - are you thinking we should make it invalid?

Reply:
It was an assumption based off the error that seemed to indicate that variants should end with a comma or newline.
I'll correct that.

Reply:
(Currently enums like `enum Fruit { Apple }` don't parse correctly, so I was not 100% sure on the correct parse)